protocol/src/main/java/tw/waterball/survive/io/protocol/scanner/ModuleClassesScanner.java

```diff
@@ -0,0 +1,81 @@
+package tw.waterball.survive.io.protocol.scanner;
+
+import java.io.File;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.util.*;
+
+public final class ModuleClassesScanner {
```

Comment:
Add an author name
and a comment describing what this class does and how to use it.